tests/unit/core/test_module_configuration.py

```diff
@@ -59,7 +59,7 @@ def test_build_in_types(self):
         params = {"int": 123, "float": 12.4, "string": "ala ma kota", "bool": True}
 
         # Check error output.
-        self.assertEqual(self.module.validate_params(params), True)
+        assert self.module.validate_params(params) == True
```

Comment:
Is there a reason we are switching this?

Reply:
Yes, please note that this test is not derived from unittest TestCase, so we do not have the access to TestCase assert methods.

This is one step towards more tight dependency on pytest, in here I did that as I started to rely on pytest 'tmpdir' fixture... honestly speaking now I am not not even sure if it was failing or not, not important I guess... the result is the same.